src/components/Post.js

```diff
@@ -0,0 +1,275 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { compose } from "redux";
+import {
+  Avatar,
+  Grid,
+  IconButton,
+  Link,
+  Menu,
+  MenuItem,
+  Paper,
+} from "@material-ui/core";
+import MoreVertIcon from "@material-ui/icons/MoreVert";
+import EmojiEmotionsIcon from "@material-ui/icons/EmojiEmotions";
+import LibraryAddIcon from "@material-ui/icons/LibraryAdd";
+import ReplyIcon from "@material-ui/icons/Reply";
+import ShareIcon from "@material-ui/icons/Share";
+import { withStyles } from "@material-ui/core/styles";
+
+const styles = {
+  root: {
+    flexGrow: 1,
+  },
+  paper: {
+    display: "flex",
+    padding: 10,
+    borderRadius: 16,
+  },
+  display: {
+    display: "flex",
+    padding: 0,
+    borderRadius: 12,
+    margin: 5,
+  },
+  content: {},
+  button: {
+    padding: 0,
+    width: 30,
+    height: 30,
+    fontSize: "0.6rem",
+    "&:hover": {
+      backgroundColor: "transparent",
+    },
+  },
+  buttongroup: {
+    margin: 0,
+    padding: 0,
+  },
+  moreGid: {
+    maxWidth: 50,
+  },
+  media: {
+    // media style
+  },
+};
+
+const menuOptions = ["edit", "delete", "report"];
+
+const user = {
+  id: 7,
+  username: "FreeBird_86",
+};
+
+class Post extends Component {
+  state = {
```

Comment:
These state attributes will need to be refactored into global store properties once we integrate Redux into our project. Fine for now though!